dotnet/src/webdriver/Chromium/ChromiumDriver.cs

```diff
@@ -234,6 +284,74 @@ public void CloseDevToolsSession()
             }
         }
 
+        /// <summary>
+        /// Returns the list of cast sinks (Cast devices) available to the Chrome media router.
+        /// </summary>
+        /// <returns>An object representing the list of available sinks.</returns>
+        public object GetCastSinks()
```

Comment:
What, exactly, does this return? We should coerce it to a proper data type, even if that means creating a new class to represent it.

Reply:
I don't know how to do that. I'm guessing something like I cast it to a collection of objects and then iterate over / map the collection?

Reply:
Go ahead and merge as-is, and I'll take a look, but yes, that's exactly what you'd do.